Content/new manuals (#27)
fictionlab/docs: docs/leo-rover/manuals/boot-and-firmware.mdx

  height="802"
/>

---

## Step 2

<h3>Connect to Leo Rover by following this tutorial:</h3>

## <LinkButton docId="leo-rover/guides/connect-to-rover-ap" />

---

## Step 3

<h3>Flash the new firmware by referring to:</h3>

## <LinkButton docId="leo-rover/guides/firmware-update" />

---

## Step 4

> <h3>
>   The rover should be fully functional now.\ To test it access the Leo Rover UI
>   and drive a little.
> </h3>

Reviewer: Prettier does not seem to work with `\n` line breaks inside a header. Maybe instead of a header, a `success` note would be more appropriate?

```suggestion
:::success

The rover should be fully functional now. To test it, access the Leo Rover UI
and drive a little.

:::
```

Author: Done + text rearranged to fit better on the site.

  height="802"
/>

---

> ## Step 2
> 
> <h3>Connect to Leo Rover by following this tutorial:</h3>
> 
> ## <LinkButton docId="leo-rover/guides/connect-to-rover-ap" />
> 
> ---
> 
> ## Step 3
> 
> <h3>Flash the new firmware by referring to:</h3>
> 
> ## <LinkButton docId="leo-rover/guides/firmware-update" />

Reviewer 2: Do not use headers as means of highlighting text. Use `<span>` instead. Also, do not highlight `<LinkButton>` components.

```suggestion
## Step 2

<span style={{ fontSize: '1.2em', fontWeight: 'bold' }}>
  Connect to Leo Rover by following this tutorial:
</span>

<LinkButton docId="leo-rover/guides/connect-to-rover-ap" />

---

## Step 3

<span style={{ fontSize: '1.2em', fontWeight: 'bold' }}>
  Flash the new firmware by referring to:
</span>

<LinkButton docId="leo-rover/guides/firmware-update" />

```

Reviewer 2: It is confusing for indexing bots from Google etc.

Author: Done

---

## Step 4

:::success

The rover should be fully functional now. To test it, access the Leo Rover UI
and drive a little.

:::

<ImageZoom
  src="/img/robots/leo/getting-started/leo-ui.webp"
  width="1888"
  height="1080"
/>

While connected to rovers access point, open any web browser and type:

```
10.0.0.1
```

UI gives you to access to basic functionalities like:

- [x] watching rover's camera stream
- [x] teleoperation
- [x] checking battery voltage

---

> <h1> That's it! Your rover should be fully functional now.</h1>

Reviewer 2: Same as above 

```suggestion
<span style={{ fontSize: '2rem', fontWeight: 'bold' }}>
  That's it! Your rover should be fully functional now.
</span>
```
